docs/config.c

```diff
@@ -1710,6 +1710,17 @@
 ** Also see $$save_history.
 */
 
+{ "history_format", DT_STRING, "%s" },
+/*
+** .pp
+** Controls the format of the entries of the history list.
+** This string is similar to $$index_format, but has its own
+** set of \fCprintf(3)\fP-like sequences:
+** .dl
+** .dt %s .dd History match
```

Comment:
Oh!  Good idea.
How about adding `%C` for the row number?
Jumping, by typing `123<enter>` is already supported.

Reply:
I had to store the `ExpandoRecord` somewhere :D Sure, I'll add that.

Reply:
I'm a bit lost, I totally forgot how to open the history dialog, can you help me please?